CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 2.0.1 - TBD
```

Comment:
Fixed by resetting to prior commit and force-pushing to my origin. I'd normally add a commit to fix it, but nobody is going be looking through the history to figure out why the CHANGELOG patch version did or didn't get bumped. :)